clients/java/src/test/java/io/camunda/client/role/UpdateRoleTest.java

```diff
@@ -38,7 +38,7 @@ void shouldUpdateRole() {
         .join();
 
     // then
-    final GroupUpdateRequest request = gatewayService.getLastRequest(GroupUpdateRequest.class);
+    final RoleUpdateRequest request = gatewayService.getLastRequest(RoleUpdateRequest.class);
```

Comment:
Wow how did that succeed 😄 I guess it has the same properties so the mapping worked fine.